code/game/objects/items/devices/taperecorder.dm

```diff
@@ -247,7 +247,7 @@
 			playsleepseconds = mytape.timestamp[i + 1] - mytape.timestamp[i]
 		if(playsleepseconds > 14 SECONDS)
 			sleep(1 SECONDS)
-			say("Skipping [playsleepseconds] seconds of silence.")
+			say("Skipping [playsleepseconds/(1 SECONDS)] seconds of silence.")
```

Comment:
This is weirder than just dividing by 10. What you did is still a magic number, it just makes it a little harder to understand. I'd prefer the old commit's version.

Reply:
Good point. I've modified the code to just divide by 10 again.